website/components/top/Description.tsx

```diff
@@ -14,12 +14,12 @@ export const Description = ({ example }: DescriptionProps) => {
   return (
     <section className="container mx-auto py-16">
       <h2 className="w-5/6 mx-auto text-gradient text-center text-3xl font-bold md:text-4xl">
-        The great experience to create slide deck with Markdown
+        Easily create beautiful slide decks with Markdown
```

Comment:
For differentiation against other slide tools, we would like to keep using the word "experience" in a lead sentence.